internal/certdb/certdb.go

```diff
@@ -130,11 +131,11 @@ func (db *CertificateRequestsRepository) UpdateCSR(id string, cert string) (int6
 	if err != nil {
 		return 0, err
 	}
-	insertId, err := result.LastInsertId()
+	affectedRows, err := result.RowsAffected()
```

Comment:
since CSR's are unique, there will always be a single ID that was modified.

Reply:
If there are 4 csrs in the db, and we modify the first one, the one with `ID=1` then `LastInsertId` would still return 5.
There is nothing wrong with that, but I don't see us using that information.

Reply:
But yes I agree affected rows will always return `1`

Reply:
i thought it only returned that. then this is better.

Reply:
It will return the number of rows that have been modified, in this case `1`
I couldn't find a function that returns Ids. I'll take another look

Reply:
I'm ok with changing this in a different PR

Reply:
cool